sycl/test-e2e/bindless_images/sampled_fetch/fetch_1D_USM.cpp

```diff
@@ -1,8 +1,7 @@
-// REQUIRES: cuda
-// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_1d_usm
```

Comment:
Why is this aspect requirement being removed? We do perform a 1D fetch on a USM backed image in the kernel.

Additionally, I think this test should also require `aspect-ext_oneapi_bindless_images_shared_usm`, as we're allocating using `sycl::malloc_shared`.